src/schema.ts

```diff
@@ -1245,6 +1246,99 @@ export const typeDefs = gql`
     isFollowing: Boolean
   }
 
+  type ArenaCategoryV2 {
+    id: Int!
+    title: String!
+    description: String!
+    postCount: Int!
+    topicCount: Int!
+    topics: [ArenaTopicV2!]
+    isFollowing: Boolean!
+  }
+
+  type ArenaTopicV2 {
+    id: Int!
+    title: String!
+    postCount: Int!
+    created: String!
+    updated: String!
+    categoryId: Int!
+    posts: PaginatedPosts
+    isFollowing: Boolean!
+  }
+
+  type PaginatedTopics {
+    totalCount: Int!
+    page: Int!
+    pageSize: Int!
+    items: [ArenaTopicV2!]!
+  }
+
+  type PaginatedArenaUsers {
+    totalCount: Int!
+    page: Int!
+    pageSize: Int!
+    items: [ArenaUserV2!]!
+  }
+
+  type PaginatedPosts {
+    totalCount: Int!
+    page: Int!
+    pageSize: Int!
+    items: [ArenaPostV2!]!
+  }
+
+  type ArenaPostV2 {
+    id: Int!
+    content: String!
+    renderedContent: String
+    created: String!
+    updated: String!
+    owner: ArenaUserV2!
+    topicId: Int!
+    flags: [ArenaFlag!]
+  }
+
+  type ArenaFlag {
+    id: Int!
+    reason: String!
+    created: String!
+    resolved: String
+    isResolved: Boolean!
+    flagger: ArenaUserV2!
+  }
+
+  type ArenaUserV2 {
+    id: Int!
+    displayName: String!
+    username: String!
+    location: String!
+    groups: [String!]!
+  }
+
+  input ArenaUserV2Input {
+    favoriteSubjects: [String!]
+    arenaEnabled: Boolean
+    shareName: Boolean
+    arenaGroups: [String!]
+  }
+
+  type ArenaNewPostNotificationV2 {
```

Comment:
Hvorfor er `new` en del av navnet? Er det en fare for at vi må utvide dette senere til å støtte andre ting som ikke nødvendigvis er en ` newPost`?

Reply:
Det er en fare for at vi må utvide for å støtte andre type notifications om vi legger til det ja.